test/multiverse/suites/view_component/view_component_instrumentation_test.rb

```diff
@@ -0,0 +1,40 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative '../rails/app'
+
+class ExampleComponent < ViewComponent::Base
+  <<~ERB
+    <%= @title %>
+  ERB
+
+  def initialize(title:)
+    @title = title
+  end
+end
+
+class ViewComponentController < ActionController::Base
+  def index
+    render(ExampleComponent.new(title: 'Hello World'))
+  end
+end
+
+class ViewComponentInstrumentationTest < ActionDispatch::IntegrationTest
+  include MultiverseHelpers
+  setup_and_teardown_agent
+
+  def test_metric_recorded
+    get('/view_components')
+
+    assert_metrics_recorded('View/view_component/view_component_instrumentation_test.rb/ExampleComponent')
```

Comment:
The metric name looked odd to me with the .rb file path, but it does match the convention we've long had in place for `View/` based metrics:

```shell
'View/model/index.html.erb/Rendering'
'View/model/_list.html.erb/Partial'
```

Reply:
Agree! I don't love the look and thought it was a bit weird, but matches 🤷‍♀️ It is also how ViewComponent itself has chosen to do [their instrumentation](https://github.com/ViewComponent/view_component/blob/c8462c2113ca1771b681fc89eb025d8e8b685727/lib/view_component/instrumentation.rb#L16) (`self.class.identifier` is the full path including `.rb`)